src/delivery/controllers/meteorsControllers.js

```diff
@@ -0,0 +1,23 @@
+import { getMeteors } from "../../useCases/getMeteors.js";
+
+export const getMeteorsController = async (req, res, next) => {
+  const {
+    "were-dangerous-meteors": wereDangerousMeteors,
+    count,
+    date,
+  } = req.query;
+
+  try {
+    let meteors = await getMeteors({
+      date: date,
+      wereDangerousMeteors: wereDangerousMeteors === "true",
+    });
+
+    res.render("meteors.njk", {
+      meteors: meteors,
+      count: count === "true",
+    });
+  } catch (error) {
+    next(`Error fetching data from NASA: ${error.message}`);
```

Comment:
create please custom Exception with status code and message
then you catch catch it in errorHandler